ChangeLog

```diff
@@ -10,6 +10,10 @@ Changes
    * Remove some redundant code in bignum.c. Contributed by Alexey Skalozub.
    * Support cmake build where Mbed TLS is a subproject. Fix
      contributed independently by Matthieu Volat and Arne Schwabe.
+   * Allow configuring the prefix operator for the archiver tool when compiling
+     the library using the makefile. Found and fixed by Alex Hixon.
+   * Allow configuring the shared library extension by setting the DLEXT
```

Comment:
I don't see what has changed functionally regarding `DLEXT`.

Reply:
Please see my comment above. I am happy to revert the change if you prefer, alternatively I can edit the ChangeLog entry to say that DLEXT no longer overwrites the environment variable.

Reply:
Oh, ok, the _environment_ variable. That seems a bit minor to include in the changelog.

Reply:
I know, but it is still a change, so I thought I would just add it in.